Get2It/Get2It/Screens/TaskListVC.swift

```diff
@@ -188,12 +188,13 @@ extension TaskListVC {
         navigationController?.navigationBar.prefersLargeTitles = true
         
         let addBarButton = UIBarButtonItem(barButtonSystemItem: .add, target: self, action: #selector(addTaskButtonTapped))
-        navigationItem.rightBarButtonItem = addBarButton
+//        navigationItem.rightBarButtonItem = addBarButton
```

Comment:
this means every Task List won't be able to add a new task. I think we still need to keep the button for the original Task List based on the Date

Reply:
based on the conversation, we decided to hide the button

Reply:
Based on our conversation, think we're good